src/Manager.cc

```diff
@@ -1119,7 +1119,7 @@ void ManagerPrivate::LoadPlugin(const tinyxml2::XMLElement *_elem)
     systemPaths.SetPluginPathEnv("IGN_LAUNCH_PLUGIN_PATH");
 
     systemPaths.AddPluginPaths(
-      gz::common::joinPaths(homePath, ".ignition", "gazebo", "plugins"));
+      gz::common::joinPaths(homePath, ".gz", "sim", "plugins"));
```

Comment:
Can we also add the old one as a tick?

Reply:
https://github.com/gazebosim/gz-launch/pull/188/commits/8baaa82c5c2a1b59bda8cd407292460b19758223 ! (And in the other occurence too!)

Reply:
Wait actually looking at the code I believe the tick-tock for the plugin path had already been handled in https://github.com/gazebosim/gz-launch/pull/163?

Reply:
Ok, I can remove them :eyes:

Reply:
https://github.com/gazebosim/gz-launch/pull/188/commits/30911f3900fac90060d186f5e0416e2c1fa7e95a Reverted!

Reply:
So, this path is already added on line 1106. I think we don't need to make any changes to this file anymore :D

Reply:
Oops https://github.com/gazebosim/gz-launch/compare/30911f3900fac90060d186f5e0416e2c1fa7e95a..d67da3843f75ebfd153b07f57d64e46f2ec6368e

Reply:
Oh wait a second.. The ticktocking logic means the whole chunk should be removed. Sec

Reply:
https://github.com/gazebosim/gz-launch/pull/188/commits/fd1ed633754a0d622b81921bf2b5d517d976cd58 We are now in a README only PR